v2/pkg/commands/build/desktop.go

```diff
@@ -20,9 +20,10 @@ func newDesktopBuilder(options *Options) *DesktopBuilder {
 func (d *DesktopBuilder) BuildAssets(options *Options) error {
 
 	// Check assets directory exists
-	if !fs.DirExists(options.ProjectData.BuildDir) {
+	buildDir := options.ProjectData.GetBuildDir()
+	if !fs.DirExists(buildDir) {
 		// Path to default assets
-		err := buildassets.Install(options.ProjectData.Path)
+		err := buildassets.Install(buildDir)
```

Comment:
Doesn't `buildassets.Install` already add a `build` dir to the path since the `assets embed.FS` from build dir contains that root folder?

Reply:
It does not, though I can see why it looks like it would. Running `wails build` in `examples/customlayout/cmd/customlayout` works as expected.

Reply:
That's interesting, I'm confused on that with diff of the PR. Before we passed e.g. `options.ProjectData.Path = ./` and it wrote it into `./build/`, now we are passing `./build/` and it also writes it into `./build/`.

Does that method `desktopBuilder.BuildAssets` even get called?